devtools/common/src/main/java/io/quarkus/cli/commands/file/GradleBuildFile.java

```diff
@@ -81,6 +82,13 @@ private void completeBuildContent(String groupId, String version) {
         if (!buildContent.contains("apply plugin: 'io.quarkus'") && !buildContent.contains("id 'io.quarkus'")) {
             res.append(System.lineSeparator()).append("apply plugin: 'io.quarkus'").append(System.lineSeparator());
         }
+        if (!buildContent.contains("apply plugin: 'java'")) {
```

Comment:
This check is not correct, since `apply plugin: 'java'` will never be there. What I think we need to have is a check for `compileJava`

Reply:
OK, yes, makes sense. It is done in the last commit :-)